sirepo/sim_data/__init__.py

```diff
@@ -414,10 +414,11 @@ def lib_files_to_run_dir(cls, data, run_dir):
     @classmethod
     def model_defaults(cls, name):
         """Returns a set of default model values from the schema."""
+        import copy
```

Comment:
Just FYI, low level modules like this can be imported at the top, but this is fine.